src/common/array_append.cairo

```diff
@@ -32,6 +33,16 @@ impl ArrayU32AppendU256 of ArrayAppendTrait<u32, u256> {
     }
 }
 
```

Comment:
Unfamiliar user can think that this functions will add 252 bits to the array, but it makes nonsense to add 252/8=31.5 bytes to tha array and also no sense to add 252/32=7.875 u32s so it is kind-of self explanatory that it will top up to 256 and than append.

My solution is to delete this trait impl and add .into() for felt252 than append it to array, because this function can be confusing.

Tell me what do u think about it?

Reply:
It's used in 3 places and it's cleaner there when we have implementation for `felt252`. Therefore I would opt for leaving it and just add a comment that it converts to u256. wdyt?

Reply:
That is ok, let's leave a comment there that it converts to u256 internally

Reply:
Should I comment on every usage or just in implementation?

Reply:
just impl i think